swagger_to_sdk_config.json

```diff
@@ -6,7 +6,7 @@
   },
   "projects": {
     "azure_mgmt_analysis_services": {
-      "swagger": "arm-analysisservices/2016-05-16/swagger/analysisservices.json",
+      "swagger": "specification/analysisservices/resource-manager/readme.md",
```

Comment:
There is no real disadvantage. The only thing is consider a simple scenario a markdown with one file. Now the service team adds a new file (new version) and updates the tag. If we dont pass the tag, we will automatically pick it up. Else, the new changes will not be picked up until we change it. (We will do that any way in our pre release activity). In essence. the only advantage (on not passing the tags) is one less work for us. Any thoughts? @veronicagg